packages/api/db/deleteSensorData.js

```diff
@@ -0,0 +1,69 @@
+async function deleteSensorData(knex) {
```

Comment:
@smattingly can you review this migration?

Reply:
@fsayyid This seems good to me, except that it deletes from `sales_task` repeatedly. I don't know the full list of task oriented tables without looking. 

I know that the DROP TABLE statements are repeated because the first appearance is inside a transaction. But I would try to avoid that duplication by using a transaction in all cases.